styles/index.css

```
@@ -0,0 +1,41 @@
+/* font-family: 'Source Serif Pro', serif; */
+body {
+    background-image: url("https://d2kq0urxkarztv.cloudfront.net/56cb72ddf554b2fc60cb4586/2807385/upload-6c4ee6cb-7839-4f1f-9faf-0202feb0f32c.jpg?e=webp&cX=2.973887814313457&cY=0&cW=2244.052224371373&cH=1497");
+    font-family: 'Source Serif Pro', serif;
+    color:black;
+
+}
```

Centering the content on a page makes it easier to read on a page

```suggestion
body {
    background-image: url("https://d2kq0urxkarztv.cloudfront.net/56cb72ddf554b2fc60cb4586/2807385/upload-6c4ee6cb-7839-4f1f-9faf-0202feb0f32c.jpg?e=webp&cX=2.973887814313457&cY=0&cW=2244.052224371373&cH=1497");
    font-family: 'Source Serif Pro', serif;
    color:black;
    text-align: center;
}
```